### PR TITLE
Adds put/get terminology and returns uid for put.

### PR DESCRIPTION
**Pull request by YAmikep**
Hi Fabian,

As I explained in this ticket (https://github.com/hellp/saferedisqueue/issues/5), I think put/get is best suited for a queue than push/pop.
I also return the uid when you put an element in the queue since it can be useful for a reference.

Let me know what you think and if you are interested to integrate these changes.

Michael

### CODE REVIEW

**Comment by hellp** [comment]
+1

Could you please update the README – just replace all push/pop examples and add a line mentioning the aliases.

**Comment by YAmikep** [comment]
I updated the README, the test file and the **main** part.

File: saferedisqueue.py [hunk 1]
```diff
@@ -102,8 +102,12 @@ def push(self, item):
                 .hset(self.ITEMS_KEY, uid, item)\
                 .lpush(self.QUEUE_KEY, uid)\
                 .execute()
+        return uid
```
Comment (hellp) [inline saferedisqueue.py:105]: Implemented like this `put` returns a  `uuid.uuid4` object. Shouldn't we rather return the string version of it, as `get`/`pop` does?

Comment (YAmikep) [inline saferedisqueue.py:105]: Good point. It would make it consistent with what `get`/`pop` returns.

Comment (YAmikep) [inline saferedisqueue.py:105]: I would do `str(uid)` but I'm not sure whether there is a better way compatible with both Python 2 and Python 3?

**Comment by hellp** [comment]
merged manually. thanks. aaaaand 3.0.0 released, tagged, and on pypi.

**Pull request closed without merging**